ocs_ci/helpers/dr_helpers_ui.py

```
@@ -902,63 +911,130 @@ def verify_pending_cleanup_alert_firing(
+            expected_text = constants.ALERT_APPLICATION_CLEANUP_PENDING
 
-    # Expand Critical alerts section (best-effort)
-    try:
-        critical_alert = acm_obj.find_an_element_by_xpath(
-            acm_loc["critical-alert"][0]
-        ).get_attribute("aria-expanded")
-
-        if critical_alert == "false":
-            critical_alert_elem = wait_for_element_to_be_clickable(
-                acm_loc["critical-alert"]
+        def _check_drawer(timeout=30):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_check_drawer`

(ANN202)